contracts/deploy/ConstAddressDeployer.sol

```diff
@@ -80,7 +84,6 @@ contract ConstAddressDeployer {
     function _deploy(bytes memory bytecode, bytes32 salt) internal returns (address deployedAddress_) {
         if (bytecode.length == 0) revert EmptyBytecode();
 
```

Comment:
there are still changes in this file. did you check if bytecode changed before/after?

Reply:
I ran npm run build on main and on this branch with the added comments and both bytecode values are identical in the contract ABIs in the artifacts folder. Bytecode: `0x608060405...`

Reply:
the beginning of the bytecode is the same for all contracts due to some common init code. you need to compare the full bytecode or it's hash. I'm seeing a difference in the bytecode locally

Reply:
Ok that makes sense, will compare bytecode hashes in the future to be sure